src/status_im/ui/components/toolbar/styles.cljs

```diff
@@ -85,9 +87,20 @@
 
 (def item-text-white-background {:color colors/blue})
 
-;;TODO(goranjovic) - Breaks the toolbar title into new line on smaller screens
-;;e.g. see Discover > Popular hashtags on iPhone 5s
-(def ios-content-item {:position :absolute :right 40 :left 40})
+;;TODO(goranjovic) - This doesn't work at all on desktop. Review why we even need absolute position hacks
```

Comment:
Creating `react-native-desktop` issue with a small js example representing a problem would be very helpful.

Reply:
@vkjr This one is entirely `status-react` issue. At some point we used an absolute positioning hack to fix centering on some iphones. This is that issue coming back at us with vengeance.